lineapy/api/api.py

```diff
@@ -194,15 +195,15 @@ def _try_delete_pickle_file(pickled_path: Path) -> None:
             raise KeyError(f"Pickle not found at {pickled_path}")
 
 
-def _try_write_to_db(value: object) -> Path:
+def _try_write_to_db(value: object) -> str:
     """
-    Saves the value to a random file inside linea folder. This file path is returned and eventually saved to the db.
+    Saves the value to a random file inside linea folder. The file name is returned and eventually saved to the db.
```

Comment:
"The file name" is incomplete info? we lose the info about what backend was used if we do that? is that the intent?

Reply:
This is for LIN-408. We want only a relative path to the artifact_storage_directory to the database. Otherwise the artifact store cannot move at all.